napari/components/_tests/test_viewer_model.py

```diff
@@ -325,6 +325,12 @@ def test_swappable_dims():
 
     labels_data = np.random.randint(20, size=(7, 12, 10, 15))
     labels_name = viewer.add_labels(labels_data).name
+    # after set first layer to viewer the point is set to lower `extent.world`
+    # so there is no point visible on current slice. Because of this there is
+    # a need to reset position to see points from slice `[0, 0, ...]`.
+    # To pass test in swapped orders setting all points to 0 is required.
```

Comment:
a few minor suggestions to clarify this comment

```suggestion
    # after adding a layer,  the dims point is set to lower `extent.world`.
    # thus after adding the points data, the dims point is (-5.0, -5.0, -5.0, -4.0)
    # Therefore, we need to reset position to see data from slice `[0, 0, ...]`
    # as required by the tests below
```

Reply:
Thanks